### PR TITLE
List operations

### PR DESCRIPTION
**Pull request by anuyts**
List operations.

### CODE REVIEW

File: Cubical/Data/FinData/Properties.agda [hunk 1]
```diff
@@ -73,6 +81,25 @@ injSucFin : ∀ {n} {p q : Fin n} → suc p ≡ suc q → p ≡ q
 injSucFin {ℕsuc ℕzero} {zero} {zero} pf = refl
 injSucFin {ℕsuc (ℕsuc n)} pf = cong predFin pf
 
+injSucFinP : ∀ {n0 n1 : ℕ} {n : n0 ≡ n1} {p0 : Fin n0} {p1 : Fin n1}
```
Comment (felixwellen) [inline Cubical/Data/FinData/Properties.agda:84]: Could you rename the path? E.g. ```{n m : ℕ} {p : n ≡ m}``` - I guess I'm not alone to think natural number, when I read ```n```...

**Review by felixwellen — CHANGES_REQUESTED** [review]
Thanks for the contribution and I'm sorry for the long wait. It looks all good to me, I just have a small renaming request.

File: Cubical/Data/FinData/Properties.agda (hunk 1 above)
Comment (anuyts) [reply Cubical/Data/FinData/Properties.agda:84]: Well it is an `I`-indexed natural number :) But I'll rename.

Comment (felixwellen) [reply Cubical/Data/FinData/Properties.agda]: Thanks!

**Review by felixwellen — APPROVED** [review]
Thans for the contribution!

**Pull request merged**